.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: main
+on:
+  - pull_request
+  - push
+jobs:
+  main:
+    name: ${{matrix.node}}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
I used `actions/setup-node`, because TBH I don’t see why `dcodeIO/setup-node-nvm` is used in most projects.

Reply:
because names: https://github.com/dcodeIO/setup-node-nvm#set-up-node-using-nvm.
`actions/setup-node` never supported those.

Reply:
It does now, so just keep `actions/setup-node` then?

Reply:
Yeah I see. That’s good. Then we can move to `actions/setup-node` indeed!

https://github.com/dcodeIO/setup-node-nvm/issues/12